Code/GraphMol/FindStereo.cpp

```diff
@@ -326,7 +326,8 @@ StereoInfo getStereoInfo(const Atom *atom) {
       if (stereo == Atom::CHI_UNSPECIFIED) {
         switch (atom->getTotalDegree()) {
           case 4:
-            stereo = Atom::ChiralType::CHI_SQUAREPLANAR;
+            // don't assume non-tetrahedral chirality
+            stereo = Atom::ChiralType::CHI_TETRAHEDRAL;
```

Comment:
Doesn't this make exactly the opposite assumption we were making, especially because this is under a `if (getAllowNontetrahedralChirality() &&
               isAtomPotentialNontetrahedralCenter(atom)) ` check? Is there some check we can add here to distinguish between a tetrahedral (3D) and a square planar (2D) configuration?

Reply:
> Doesn't this make exactly the opposite assumption we were making, especially because this is under a `if (getAllowNontetrahedralChirality() && isAtomPotentialNontetrahedralCenter(atom)) ` check? Is there some check we can add here to distinguish between a tetrahedral (3D) and a square planar (2D) configuration?

Yes, this is making exactly the opposite assumption: previously if an atom was four coordinate and *could* have non tetrahedral stereo, we assumed that it *did* have non-tetrahedral stereo. Now we're assuming tetrahedral unless there's some information otherwise (i.e. you have to explicitly state that the thing is square planar if you want the RDKit to treat it that way).

Given the relevant prevalence if tetrahedral geometries in the systems the RDKit is usually used with, I think defaulting to tetrahedral is better.

Without actually looking at coordinates, I don't think we can make an informed decision.

Reply:
OK, let's make this assumption then. We can always come back to this if we find it problematic.